docs/gatsby-config.js

```diff
@@ -42,75 +42,74 @@ const themeOptions = {
+              'collect/collectd',
               'collect/copy',
               'collect/copy-from-host',
-              'collect/data',
               'collect/exec',
               'collect/http',
-              'collect/logs',
-              'collect/run-pod',
-              'collect/collectd',
               'collect/configmap',
               'collect/secret',
-              'collect/postgresql',
+              'collect/longhorn',
               'collect/mssql',
               'collect/mysql',
+              'collect/logs',
+              'collect/postgresql',
               'collect/redis',
-              'collect/ceph',
-              'collect/longhorn',
               'collect/registry-images',
+              'collect/run-pod',
+              'collect/data',
               'collect/sysctl',
             ],
             "Redact": [
               "redact/index",
               "redact/redactors",
-              "redact/database-connection-strings",
+              "redact/api-tokens",
               "redact/aws-credentials",
               "redact/generic-connection-strings",
+              "redact/database-connection-strings",
               "redact/ip-addresses",
               "redact/passwords",
-              "redact/api-tokens",
               "redact/usernames",
             ],
             Analyze: [
               'analyze/index',
               'analyze/outcomes',
+              'analyze/ceph-status',
+              'analyze/cluster-pod-statuses',
               'analyze/cluster-resource',
               'analyze/cluster-version',
-              'analyze/cluster-pod-statuses',
-              'analyze/distribution',
               'analyze/container-runtime',
-              'analyze/node-resources',
+              'analyze/crd',
               'analyze/deployment-status',
-              'analyze/statefulset-status',
-              'analyze/replicaset-status',
-              'analyze/job-status',
               'analyze/image-pull-secrets',
               'analyze/ingress',
-              'analyze/storage-class',
-              'analyze/configmap',
-              'analyze/secrets',
-              'analyze/crd',
-              'analyze/regex',
+              'analyze/job-status',
               'analyze/json-compare',
-              'analyze/yaml-compare',
-              'analyze/postgresql',
+              'analyze/distribution',
+              'analyze/secrets',
+              'analyze/longhorn',
               'analyze/mssql',
               'analyze/mysql',
-              'analyze/ceph-status',
-              'analyze/longhorn',
+              'analyze/node-resources',
+              'analyze/postgresql',
+              'analyze/regex',
+              'analyze/replicaset-status',
+              'analyze/storage-class',
+              'analyze/configmap',
+              'analyze/yaml-compare',
               'analyze/registry-images',
               'analyze/weave-report',
+              'analyze/statefulset-status',
```

Comment:
there's a lot of re-arrangement here, is this intended?

Reply:
> there's a lot of re-arrangement here, is this intended?

Yes. I rearranged the TOC to have it in alphabetical order.

Reply:
if that's a pattern used throughout, makes sense